ufm-stream-to-fluentd/ufm-stream-to-fluentd.py

```diff
@@ -0,0 +1,213 @@
+import requests
```

Comment:
mellanox copyrights

Reply:
@mike-dubman which license we should use? apache?

Reply:
* https://github.com/openucx/ucx/blob/master/LICENSE#L2
keep only mellanox line, rest orgs can remove

* file header should be as github.com/openucx/ucx/blob/master/examples/ucp_client_server.c#L1-L5 with correct year